src/token/types.ts

```diff
@@ -13,9 +13,32 @@ export type Token = {
   symbol: string; // A ticker symbol or shorthand, up to 11 characters
 };
 
+// The raw response from the Swap API
+export type RawTokenData = {
+  name: string;
+  address: Address;
+  currencyCode: string;
+  decimals: number;
+  imageURL: string;
+  blockchain: string;
+  aggregators: string[];
+  swappable: boolean;
+  unverified: boolean;
+  chainId: number;
+};
+
 /**
  * Note: exported as public Type
  */
 export type TokenChipReact = {
   token: Token;
 };
+
+/**
+ * Note: exported as public Type
+ */
+export type GetTokensOptions = {
+  limit?: string;
+  search?: string;
+  page?: string;
```

Comment:
Is search a filter for both `address`, `name` and `symbol`?

Reply:
yea, looks like it works

```
const resp = await getTokens({ limit: '1', search: '0x4ed4e862860bed51a9570b96d89af5e1b0efefed' });
```

```
onchainkit % npx tsx src/examples/swap.ts
[
  {
    address: '0x4ed4e862860bed51a9570b96d89af5e1b0efefed',
    chainId: 8453,
    decimals: 18,
    image: 'https://d3r81g40ycuhqg.cloudfront.net/wallet/wais/3b/bf/3bbf118b5e6dc2f9e7fc607a6e7526647b4ba8f0bea87125f971446d57b296d2-MDNmNjY0MmEtNGFiZi00N2I0LWIwMTItMDUyMzg2ZDZhMWNm',
    name: 'DEGEN',
    symbol: 'DEGEN'
  }
]
```